docs/src/reference/asciidoc/index.adoc

```diff
@@ -27,18 +27,26 @@
 
 {ehtm} is an `umbrella' project consisting of three similar, yet independent sub-projects with their own, dedicated, section in the documentation:
 
-{eh} proper:: interact with {es} from within a Hadoop environment. If you are using {mr}, Hive, Pig, {sp}, {st}, or Cascading, this project is for you. For feature requests or bugs, please open an issue in the https://github.com/elastic/elasticsearch-hadoop/issues[Elasticsearch-Hadoop repository].
+{eh} proper::
+Interact with {es} from within a Hadoop environment. If you are using {mr}, Hive, Pig, {sp}, {st}, or Cascading, this
+project is for you. For feature requests or bugs, please open an issue in the
+https://github.com/elastic/elasticsearch-hadoop/issues[Elasticsearch-Hadoop repository].
 
-repository-hdfs:: use HDFS as a back-end repository for doing snapshot/restore from/to {es}. For more information, refer to its https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-hdfs.html[home page]. For feature requests or bugs, please open an issue in the https://github.com/elastic/elasticsearch/issues[Elasticsearch repository] with the ":Plugin Repository HDFS" tag.
+repository-hdfs::
+Use HDFS as a back-end repository for doing snapshot/restore from/to {es}. For more information, refer to its
+https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-hdfs.html[home page]. For feature requests or
+bugs, please open an issue in the https://github.com/elastic/elasticsearch/issues[Elasticsearch repository] with the
+":Plugin Repository HDFS" tag.
 
-{ey}:: run {es} on top of YARN - see <<es-yarn>>. This project is in beta.
+{ey} [DISCONTINUED]::
```

Comment:
@dedemorton thanks for the input. The rationale behind having it persistent here is the fact that the project has been around for a couple years at this point, so we'd like to keep this discontinuation visible likely through the 6.x releases before removing it completely in 7.0.

Does that sound reasonable?